File: codeforces/stringPresent.cpp
#include <bits/stdc++.h>
using namespace std;
string f(string s, string t)
{
    int n = s.size();
    int m = t.size();
    string ans = "";
    // map<char, int> mp;
    // string ans = "";
    // for (int i = 0; i < m; i++)
    // {
    //     mp[t[i]]++;
    // }
    // for (int i = 0; i < n; i++)
    // {
    //     if (mp.find(s[i]) == mp.end())
    //     {
    //         ans += s[i];

    //     }
    // }
    // return ans;
    for (int i = 0; i < n; i++)
    {
        // bool flag = false;
        for (int j = 0; j < m; j++)
        {
            if (s[i] == t[j])
            {
                // flag = true;
                break;
            }
            if (j == m - 1)
            {
                ans += s[i];
            }
        }
    }
    return ans;
}

int main()
{
    string s = "tiger";
    string t = "er";
    cout << f(s, t);
}

File: codeforces/stringpresent.java
import java.util.*;

class StringPresent {
    public static String present(String s1, String s2) {
        int n = s1.length();
        int m = s2.length();
        String ans = "";
        for (int i = 0; i < n; i++) {
            for (int j = 0; j < m; j++) {
                if (s1.charAt(i) != s2.charAt(j)) {
                    ans += s1.charAt(i);
                    break;
                }
            }
        }
        return ans;
    }

    public static void main(String[] args) {
        String s1 = "tiger";
        String s2 = "er";
        System.out.println(present(s1, s2));

    }
}